### PR TITLE
 Text styles in no results page are incorrect #1588 

### LINKED ISSUE #1588: Text styles in no results page are incorrect
## Description
Title and description texts in the no results page do not follow the styles defined in [the mockup](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1479%3A20842). H1 should be `Heading 4` and the text below be `Description regular`.

## Reproduction
1. Open openverse on mobile or within the smallest breakpoint.
2. Search for "asd32asda"
3. Notice that title and description texts, above source links, are bigger than mockups.

## Screenshots
|Implementation|Design|
|:---|:---|
|CleanShot 2022-07-26 at 14 36 56@2x|CleanShot 2022-07-26 at 14 35 21@2x|

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by kickeyhickey**
## Fixes

Fixes #1588 by @panchovm

## Description

On component VNoResults.vue
  - changed h5 to h4 below <div class="no-results text-center md:text-left">
  - deleted text-3xL -  on <h6 class="mt-10 text-3xl font-normal md:mt-16 md:font-bold">
  
Screenshot from 2022-08-02 12-36-33

## Testing Instructions
In live build
   - search "fjkdls;a;"
   - when you arrive at 'no results' page
        - inspect page, view on 'mobile view' to see text size changes

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — CHANGES_REQUESTED** [review]
Thank you for your quick PR, @kickeyhickey!

It looks fine, but in light of the accessibility requirements described in #1614 (that we need to fix all over the site), could you please replace the heading elements:
- `h4` (with `$t('no-results.heading')` should become `h1`
- `h6` should become `h2`

The headings should have the same styles as the [mockup](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1479%3A20842), so you would need to add some Tailwind classes. You can find the Tailwind styles on the [Openverse Tailwind preview page](https://wordpress.github.io/openverse-frontend/tailwind/#Font%20Sizes).
Note: In Figma, there's an `Inspect` tab that you can use to view the font sizes and other properties of elements. However, you can only see this tab in your own documents, not the ones shared with you. To overcome this problem and to see the `Inspect` tab for the Openverse mockups, you can make your own copy of them.

For the `h1`, Figma designs show it should have a size of 32px on the screens that are wider than `md` breakpoint, and 24px for smaller screens. So you would need to add `text-6xl md:text-4xl ` class to it.
For the `h2`, Figma designs show it should have a size of 19px for `md` and above, and 14px for below `md` screens, so you would need to add `text-base md:text-3xl` to it.

**Comment by kickeyhickey** [comment]
@obulat We went ahead and implemented your changes. Thank you for your detailed reply. We want our code to be as accurate as possible. Still learning!

File: src/components/VErrorSection/VNoResults.vue
Comment (obulat) [inline src/components/VErrorSection/VNoResults.vue:3]:
```suggestion
    <h1 class="break-words text-4xl md:text-6xl">
```
Sorry for mixing the sizes up :)

**Comment by kickeyhickey** [comment]
@obulat We went ahead and implemented your change! Thanks for your help.

**Review by krysal — APPROVED** [review]
This is looking good @kickeyhickey! Thank you for your contribution, I noticed a tiny style needed but leaving an approval to partially unblock. We merge PRs with at least 2 approvals, so Olga can check again and merge it once is only again.

File: src/components/VErrorSection/VNoResults.vue
Comment (obulat) [reply src/components/VErrorSection/VNoResults.vue:6]: This is great, @kickeyhickey! I will approve and merge after you commit @krysal's suggestion.

**Review by panchovm — COMMENTED** [review]
It looks great ✨ 

I just noticed the description text is incorrect; it says `Try a different query or use one of the additional sources to expand your search` when it should say `Try a different search or see the additional sources`.

It might require creating another PR but notice it in case you can apply the change.

**Comment by kickeyhickey** [comment]
@panchovm Hey! I went ahead and made the changes that @krysal suggested. We found the incorrect text in the en.JSON file. I can go ahead and make that change for you on this ticket. I also don't mind opening another Issue/ PR for it if the team deems it so. 
CC: @obulat

**Comment by zackkrida** [comment]
@kickeyhickey a standalone PR for that would be _excellent_; thank you!

**Review by zackkrida — APPROVED** [review]
LGTM!

**Comment by zackkrida** [comment]
I pushed a small fix to the tailwind class order so that linting would pass. Nice work!

**Pull request merged**
